### PR TITLE
[ZH] Update `People\The_Team\Beatmap_Nominators`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] Changes on #7111 have apply to this PR

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Nominators/zh.md [hunk 1]
```diff
@@ -2,40 +2,43 @@
...
 ---
 
-# Beatmap Nominators
+# 谱面审核成员 (BN)
 
-*另请参阅：[Beatmap Nominator 规则](Rules), [Beatmap Nominators 常规信息](General_Information)，和 [成为 Beatmap Nominator](Becoming_a_Beatmap_Nominator)*
+*请参阅：[BN 规章制度](Rules)、[对 BN 的期望](Expectations)、[BN 须知](General_Information)、[成为 BN](Becoming_a_Beatmap_Nominator)*
 
-Beatmap Nominators 通常缩写为 *BN*, 是负责[提名谱面](/wiki/Beatmap_ranking_procedure)的小组。自2014年8月22日起，他们不再是员工的一部分，在游戏、聊天室以及论坛中取消了他们的权限和红名。自2015年2月1日起，团队名称从 *Beatmap Appreciation Team* 更改为 *Beatmap Nominators*。目前，他们在论坛和谱面讨论中的名称为紫色。
+**谱面审核成员** 全称为 Beatmap Nominators (***BN***)，是负责[谱面审核流程](/wiki/Beatmap_ranking_procedure)的一群玩家。他们在论坛和[谱面讨论区](/wiki/Beatmap_discussion)标记为紫色用户名。
```
[the remark this answers is not in the document]
Comment (RushFTK) [reply wiki/People/The_Team/Beatmap_Nominators/zh.md:11]: 此外，考虑到英文原文使用"Members of the Beatmap Nominators"，称BN为谱面审核**团队**是否更合适一些呢？这个也是可选的。

File: wiki/People/The_Team/Beatmap_Nominators/zh.md [hunk 2]
```diff
@@ -2,40 +2,43 @@
...
+除此之外，所有 BN 都应发扬[对 BN 的期望](Expectations)这篇文章所阐述的精神：不拘泥于现状，努力提升自己；将工作重心放在各种各样的谱师或谱面上，为更大的社区服务。
 
-除了常规的谱面摸图和 Rank 外，Beatmap Nominators 的正式成员还负责取消在合格阶段中发现有问题的谱面的资格。如果 BN 正式成员有兴趣将来加入 Nomination Assessment Team，他们也可以提交 NAT 的评估申请。
+BN 分为两类：正式 (Full) 成员和见习 (prob) 成员。
 
-大约每三个月，BN 正式成员的提名活动以及质量将由 Nomination Assessment Team 进行评估。如果他们的提名由于必不可少的原因而经常被重置，或者他们的行为令人无法接受，那么他们可能会转到见习成员当中。
+### 正式 (Full) 成员
 
-低于最低提名活跃度标准的正式成员将不会进入见习期，但是如果他们没有做出改善，他们将会被从团队中移除。
+除了平常的摸图和上架谱面之外，正式成员还负责下架某些已发现问题的，处于过审 (Qualified) 状态的谱面。如果他们想要加入审核评估团队 (NAT)，也可以提交相应的评估申请。
```
Comment (RushFTK) [inline wiki/People/The_Team/Beatmap_Nominators/zh.md:23]:
```suggestion
除了平常的摸图和上架谱面之外，正式成员还负责下架某些已发现问题的，处于过审 (Qualified) 状态的谱面。如果他们有意在未来加入审核评估团队 (NAT)，也可临时承担甄选BN申请的工作。
```
**你必须修改这条。**

> Full Beatmap Nominators may also participate in occasional Beatmap Nominator application assessment if they are interested in joining the Nomination Assessment Team in the future.

原文的意思是承担一部分NAT的工作，也就是去审BN申请这一工作，如果说提交相应的评估申请，更像是填写一个加入NAT的表格，我认为这里的翻译会造成含义的偏差。

File: wiki/People/The_Team/Beatmap_Nominators/zh.md [hunk 3]
```diff
@@ -2,40 +2,43 @@
...
 
-### 见习成员
+大约每三个月（90 天），[审核评估团队 (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) 会[评估](/wiki/People/The_Team/Nomination_Assessment_Team/Evaluations)正式成员的行为规范、提名活跃度、提名质量。如果他们的提名经常因犯了明显的错误而被重置，亦或是某些行为不可接受，则他们可能会被降级为见习成员。
+
+如果正式成员达不到最低活跃度要求，并在警告后没有采取相应的补救措施，则他们会被直接从 BN 中开除，而不是降级。
+
+### 见习 (prob) 成员
 
-见习期是一种可以用于监视新的 BN 或者可以更密切地关注 BN 的方式。作为见习 BN 成员时，他们的主要工作是通过通过在 Rank 系统中来专门推广谱面，来建立 BN 的基础。每个谱面 Rank 过程中必须至少有一位正式 BN 参与，因此见习成员不能对一个只收到其他见习成员提名的谱面进行最终提名。他们也不能提名混合模式谱面集。
+见习期能够用来帮助 NAT 监督或关注新 BN 的行为。见习成员的主要职责是打好 BN 的基础，并学习将谱面推至上架周期。谱面若要上架 (Ranked)，则提名者中必须有至少一位正式成员。因此，见习成员不能提名已被其他相同游戏模式的见习成员提名过的谱面。
```
Comment (RushFTK) [inline wiki/People/The_Team/Beatmap_Nominators/zh.md:31]:
```suggestion
设立见习期的目的，是用来帮助 NAT 关注新的或者待定的 BN 的行为。见习成员的主要职责是在实际推送铺面上架流程中打好作为 BN 的基本功。谱面若要上架 (Ranked)，则提名者中必须有至少一位正式成员。因此，见习成员不能提名已被其他相同游戏模式的见习成员提名过的谱面。
```
> Probation is used to monitor new or concerning Beatmap Nominators more closely. The main role for probationary Beatmap Nominators is to establish a foundation as a Beatmap Nominator through exclusively promoting beatmaps in the ranking process. At least one full nominator must be involved in the ranking process for each game mode of a beatmap, so probationary members cannot give the final nomination to a beatmap which has only received nominations from other probationary members for their game mode.

1. 第一句，首先英文原文只有一个monitor，所以译文非必要时用一个词就够了。其次不是新BN才是见习的，还有那些记大过的BN，我不确定concerning 翻译成待定还是记过比较好。
2. 第二句，through，所以这两个不是并列学习的内容，而是通过手段学习一个内容。

File: wiki/People/The_Team/Beatmap_Nominators/zh.md [hunk 4]
```diff
@@ -2,40 +2,43 @@
...
 
-BN 的新成员将从一个月的见习期开始。如果他们的提名和行为令人满意，他们将晋升为 BN 正式成员。否则，他们将被继续审核一个月或从团队中移除。
+新加入的 BN 都会有一个月的见习期。如果他们的提名顺利，行为合规，则他们会在[评估](/wiki/People/The_Team/Nomination_Assessment_Team/Evaluations)通过后晋升为正式成员。反之，他们的见习期会再加一个月，甚至被开除。
 
-当 BN 被置于见习期时，他们无法因为相同的原因再次被置入见习期。比如：一名 BN 因为不良行为而被置入见习期，如果他们再次表现出相同的不良行为，即使他们在正式成员名单中，也会被直接从团队内移除。
+如果 BN 因为犯下错误被降级成见习成员，随后他们再次犯下了类似的错误，则会被开除。比如，某位 BN 因为不良行为而被降级成见习成员，无论他们是否因度过见习期而返回正式成员，只要再犯，就会被直接开除。
```
Comment (RushFTK) [inline wiki/People/The_Team/Beatmap_Nominators/zh.md:35]:
```suggestion
因犯错而被降级为见习成员的BN，同季度中二犯同种错误将会导致其被开除。比如，某位 BN 因为不良行为而被降级成见习成员，无论他们是否因度过见习期而返回正式成员，只要再犯，就会被直接开除。
```

**你必须修改这条。**

> When a Beatmap Nominator is placed on probation, they cannot be placed on probation again for the same reason. For example, if a Beatmap Nominator is placed on probation for poor behaviour, they will be removed from the Beatmap Nominators if they exhibit the same poor behaviour again, even if they are a full Beatmap Nominator during the second infringement.

你可以对比一下缺了哪些信息，自行调整语序。

**Review by RushFTK — CHANGES_REQUESTED** [review]

**Comment by Walavouchey** [comment]
@RushFTK should the changes here be applied?

**Comment by RushFTK** [comment]
@Walavouchey nope, some of them must be fixed, so i will wait him response.

File: wiki/People/The_Team/Beatmap_Nominators/zh.md (hunk 4 above)
Comment (RushFTK) [reply wiki/People/The_Team/Beatmap_Nominators/zh.md:35]: 现在看这里我好像搞错了。。。（）

File: wiki/People/The_Team/Beatmap_Nominators/zh.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/People/The_Team/Beatmap_Nominators/zh.md:11]: 你这个应该是指BNG(Group)，~~当然英文原文也很容易弄混这两个概念~~，还是保留吧
keep

File: wiki/People/The_Team/Beatmap_Nominators/zh.md (hunk 4 above)
Comment (YumeMuzi) [reply wiki/People/The_Team/Beatmap_Nominators/zh.md:35]: wtf

Comment (YumeMuzi) [reply wiki/People/The_Team/Beatmap_Nominators/zh.md:35]: 这里意思是，不管BN因为什么内在原因，只要再次犯相同类型的严重错误，不管是在PBN还是恢复正式BN，都会被下狗。
regardless of BN intentions or status (full/prob), if they make the same type of serious mistake again, they will be resigned.

**Comment by YumeMuzi** [comment]
also, do I need to update the BN list of changes during this PR period?

**Review by RushFTK — APPROVED** [review]
fine for me in translation part.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**